code/modules/food_and_drinks/food/snacks_vend.dm

```diff
@@ -13,6 +13,32 @@
 	tastes = list("candy" = 1)
 	foodtype = JUNKFOOD | SUGAR
 
+/obj/item/reagent_containers/food/snacks/candy/bronx
+	name = "South Bronx Paradise bar"
+	desc = "Lose weight, guaranteed! Wait, the package actually says 'South Bronx Parasite'..."
+	icon_state = "bronx"
+	item_state = "candy"
+	trash = /obj/item/trash/candy
+	list_reagents = list(/datum/reagent/consumable/nutriment = 4, /datum/reagent/consumable/sugar = 2, /datum/reagent/yuck = 1)
+	junkiness = 10
+	bitesize = 10
+	filling_color = "#e4d4b7"
+	tastes = list("candy" = 5, "weight loss" = 4, "insect larva" = 1)
+	foodtype = JUNKFOOD | RAW | GROSS
+
+/obj/item/reagent_containers/food/snacks/candy/bronx/On_Consume(mob/living/eater)
+	. = ..()
+	if(ishuman(eater))
+		var/mob/living/carbon/human/carl = eater
+		var/datum/disease/P = new /datum/disease/parasite()
+		carl.ForceContractDisease(P, FALSE, TRUE)
+
+/obj/item/reagent_containers/food/snacks/candy/bronx/examine(mob/user)
+	. = ..()
+	to_chat(user, "<span class='notice'>Geeze, you need to get to get your eyes checked. You should look again...</span>")
+	desc = "Lose weight, guaranteed! Caramel Mocha Flavor! WARNING: PRODUCT NOT FIT FOR HUMAN CONSUMPTION. CONTAINS LIVE DIAMPHIDIA SPECIMENS."
+	name = "South Bronx Parasite bar"
```

Comment:
have some kind of flag for this so it's not being spammed/updating name every time